test/clt-tests/tables-interaction/test-tables-interaction.rec

```diff
@@ -0,0 +1,91 @@
+––– block: ../base/dind/init –––
+––– input –––
+mkdir /tmp/local-index
+––– output –––
+––– input –––
+mkdir /tmp/dist-index
+––– output –––
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:latest > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:6.2.12 > /dev/null; echo $?
```

Comment:
We need to compare latest release with the current dev, not latest release with itself or another release.

Reply:
Done